axon-5/api-changes.md

```diff
@@ -14,9 +14,70 @@ Major API Changes
   recommended resolution path in the JavaDoc. It is strongly recommended to (1) upgrade to the latest Axon Framework 4
   version, (2) adjust any deprecations from Axon Framework you are using as recommended, and then (3) to make the change
   towards Axon Framework 5.
+* The entire API of the `UnitOfWork` has been rewritten to (1) construct an " async-native" flow to support both an
```

Comment:
Verry nitty, but I think single quotes are better, and the space before `async-native` should be removed.

Reply:
Ah sharp, thanks. And agreed, single quotes  are better suited here. Thanks for spotting, and the review! :raised_hands: